scripts/ci-build-kubernetes.sh

```diff
@@ -112,7 +112,10 @@ main() {
             docker tag "${OLD_IMAGE_URL}" "${NEW_IMAGE_URL}" && docker push "${NEW_IMAGE_URL}"
         done
 
+        echo "Uploading binaries to Azure storage container ${JOB_NAME}"
+
         for BINARY in "${BINARIES[@]}"; do
+            echo "uploading ${KUBE_GIT_VERSION}/bin/linux/amd64/${BINARY}"
```

Comment:
nit: slight preference for assigning `${KUBE_GIT_VERSION}/bin/linux/amd64/${BINARY}` to a var for re-use

Reply:
updated